src/api/app/models/staging/staging_project.rb

```diff
@@ -8,6 +8,21 @@ def staging_identifier
       name[/.*:Staging:(.*)/, 1]
     end
 
+    def classified_requests
+      requests = (requests_to_review + staged_requests.includes(:reviews)).uniq.map do |request|
```

Comment:
As I said here: https://github.com/openSUSE/open-build-service/pull/6224#discussion_r232967346 NOO!!!!!

@bgeuken is this also going to be removed?

Reply:
Yes, we can drop it :thumbsup:

Reply:
Noooooo... haaaaaaaaaaaaaaaaaaaa

![hair-on-fire](https://user-images.githubusercontent.com/1102934/48545009-39292800-e8c5-11e8-8ec9-290446f7cab3.gif)